source/gui/include/scanform.h

```diff
@@ -9,39 +9,48 @@
 #include <muondetector_structs.h>
 
 // list of possible scan parameters
-const static QVector<QString> SP_NAMES = { "VOID", "THR1", "THR2" };
+const static QVector<QString> SP_NAMES = { "VOID", "THR1", "THR2", "BIAS", "DAC4" };
 // list of possible observables
-const static QVector<QString> OP_NAMES = { "VOID", "UBXRATE" /*, "GPIORATE"*/ };
```

Comment:
I would leave the commented-out GPIORATE here, since this would be the next scan observable to be implemented and is therefore helpful as a reminder

Reply:
Instead, use issues on github.

Stumbling on this comment will only happen by coincidence, and it is not clear why this is commented out or what it is supposed to mean.

If comments are used as a reminder, use a comment in the form of
`//TODO: Implement GPIORATE scan observable`
Those can be scanned and displayed by an IDE or other tools and offer useful information.